packages/client/hmi-client/src/utils/log-buffer.ts

```diff
@@ -0,0 +1,83 @@
+// import API from '@/api/api';
+import { useToastService } from '@/services/toast';
+import axios from 'axios';
+import useAuthStore from '../stores/auth';
+
+const LOGS = axios.create({
+	baseURL: '/api'
+});
+
+// Hook in bearer tokens
+// See
+// - https://medium.com/swlh/handling-access-and-refresh-tokens-using-axios-interceptors-3970b601a5da
+LOGS.interceptors.request.use(
+	(config) => {
+		const auth = useAuthStore();
+
+		if (config.headers) {
+			config.headers.Authorization = `Bearer ${auth.token}`;
+		} else {
+			config.headers = { Authorization: `Bearer ${auth.token}` };
+		}
+		return config;
+	},
+	(error) => {
+		console.warn(error);
+	}
+);
+
+const toast = useToastService();
+
+// LogDetails Interface
+interface LogDetails {
```

Comment:
Now we end types with the word `Type`, [see documentation](https://github.com/DARPA-ASKEM/TERArium/blob/main/packages/client/hmi-client/DEVELOPMENT.md#types)

Reply:
Yeah I remember this but maybe my comment about thinking it's weird to force `Interface` usage and `Type` nomenclature lol.  Will change it to `Type` cause I don't have big feelings over it :)